akka-http-core/src/test/scala/akka/http/impl/engine/server/HttpServerTestSetupBase.scala

```diff
@@ -71,4 +71,13 @@ abstract class HttpServerTestSetupBase {
   def send(string: String): Unit = send(ByteString(string.stripMarginWithNewline("\r\n"), "UTF8"))
 
   def closeNetworkInput(): Unit = netIn.sendComplete()
+
+  def shutdownBlueprint(): Unit = {
+    netIn.sendComplete()
+    requests.expectComplete()
+
+    responses.sendComplete()
+    netOut.expectBytes(ByteString("HTT")) // ???
```

Comment:
Well, looks like beginning of the `HTTP/1.1` response... most likely this should not happen but I have not read the sources in detail here yet